prime-router/src/testIntegration/resources/datatests/mappinginventory/xpn/xpn-to-humanname-xpn10-populated-xpn12-13-empty.hl7

```diff
@@ -0,0 +1,3 @@
+MSH|^~\&#|||||20230501102531-0400||ORU^R01^ORU_R01|3003786103_4988249_33033||2.5.1|||||USA|UNICODE UTF-8
+PID|1||11102779^^^^MR||SMITH^BB SARAH^^^^^L
```

Comment:
In this and the other xpn test, can we remove PID.3, MSH.17, and MSH.18? Will require FHIR updates as well, but will make these be a little bit less affected by future updates to CX/MSH.